pkg/prowgen/prowgen_test.go

```diff
@@ -469,6 +469,21 @@ func TestGenerateJobs(t *testing.T) {
 				Repo:   "repository",
 				Branch: "branch",
 			}},
+		}, {
+			id: "kvm label",
+			config: &ciop.ReleaseBuildConfiguration{
+				Resources: map[string]ciop.ResourceRequirements{
+					"*": {Requests: ciop.ResourceList{"devices.kubevirt.io/kvm": "1"}},
```

Comment:
very clever, or too clever? only time will tell

Reply:
Not sure if I get your point.
That is how our user uses it for now.

https://github.com/openshift/release/blob/0f651ff1091687662a4799e57f5455504a45a4d3/ci-operator/config/openshift/os/openshift-os-master.yaml#L13

I cannot see what it means to build pods though.
Is it what you think only time will tell?

Reply:
avoid to apply to build pods: the 2nd use case
https://github.com/openshift/release/blob/f896da9216d4742ce591f04fad80331dfc8f47c4/ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml#L26-L37